priv/posts/marketing/20240530023712_getting-to-kow-natalia-chechina.md

```diff
@@ -0,0 +1,80 @@
+{
+    "title": "Getting to know - Natalia Chechina",
+    "authors": ["The Marketing Working Group"],
+    "slug": "getting-to-know-natalia-chechina",
+    "category": "marketing",
+    "tags": ["developer", "marketing", "getting-to-know-us", "bio"],
+    "datetime": "2024-06-01T02:37:12.810583Z"
+
+}
+
+---
+Getting to know to Natalia Chechina
+---
+
+#  Getting to know - Natalia Chechina
+
+*Welcome to another edition in our Getting to Know Us series. Getting to know us consists of articles and interviews about a wide variety of people in our ecosystem, with the main goal of making their work with BEAM technologies visible. Some have been around a long time, while others may seem new to you, but they all share a love for this ecosystem. If you know of interesting work that deserves more visibility, let us know at marcom@erlef.org!*
+
+<img src="/images/getting-to-know/natalia-chechina.png" class="img-fluid" alt="Natalia-Chechina"/>
+
+Natalia Chechina is a developer at Erlang Solutions specializing in fintech.  Her research interests include distributed computation, robotics, parallel programming, stochastic modeling, and autonomous systems. She was the chair of the Steering Committee at Erlang Workshop in 2021–2023 and serves as an industrial supervisor, providing valuable guidance and insights to students working on Msc and PhD projects.  She is also a Chair of the Embedded Working Group of our Foundation. Today we get to know her.
+
+#### How did you decide to learn programming and what were your first steps on it?
+
+Programming was a practical choice for me. I was really good at math at school. While I was participating in all sorts of competitions, including international ones, I faced the question of what path to pursue. Initially, I wanted to be a pure math engineer, but since there were not many appealing job opportunities I could see at that age and environment I decided to go for programming – it’s really close to math, so it was relatively easy to learn.

```

Comment:
```suggestion
Programming was a practical choice for me. I was really good at math in school. While I was participating in all sorts of competitions, including international ones, I faced the question of what path to pursue. Initially, I wanted to be a pure math engineer, but since there were not many appealing job opportunities I could see at that age and environment I decided to go for programming – it’s really close to math, so it was relatively easy to learn.

```